src/test/java/org/aksw/simba/squirrel/data/uri/filter/RDBKnownUriFilterTest.java

```diff
@@ -0,0 +1,60 @@
+package org.aksw.simba.squirrel.data.uri.filter;
+
+import com.rethinkdb.RethinkDB;
+import junit.framework.TestCase;
+import org.aksw.simba.squirrel.RethinkDBMockTest;
+import org.aksw.simba.squirrel.data.uri.CrawleableUri;
+import org.aksw.simba.squirrel.model.RDBConnector;
+import org.junit.Assert;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.UnknownHostException;
+import java.util.List;
+
+
+public class RDBKnownUriFilterTest extends TestCase {
```

Comment:
Tests are always good. However, using `extends TestCase` is outdated. Everything that starts with `junit.framework.` is outdated :wink: 

With newer JUnit versions, you simply add `@Test` to the methods that test something. The `setUp` method should get `@before` and the `tearDown` method with `@After`.